tests/fault_tolerance/deploy/test_trtllm_cancel_burst_recovery.py

```
@@ -0,0 +1,677 @@
+                    worker_metrics_urls=worker_urls,
+                    label="after_fresh_probe",
+                )
+                _assert_trtllm_metrics_available(
+                    fresh_metrics, cycle_dir / "fresh_probe"
+                )
+        finally:
+            port_forwards = {"Frontend": port_forward, **worker_port_forwards}
+            for label, pf in port_forwards.items():
+                try:
+                    pf.stop()
+                except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)